common/app/layout/Breakpoint.scala

```diff
@@ -0,0 +1,19 @@
+package layout
+
+import scalaz.syntax.std.option._
+
+sealed trait Breakpoint {
+  val minWidth: Option[Int]
+}
+
+case object Mobile extends Breakpoint {
+  val minWidth = None
+}
+
+case object Tablet extends Breakpoint {
+  val minWidth = 740.some
```

Comment:
Don't worry @robertberry , even if it is you, I wouldn't kill you. I'd just shoot you in the balls.